lib/ae_mdw/names.ex

```diff
@@ -338,6 +338,109 @@ defmodule AeMdw.Names do
     )
   end
 
+  @spec fetch_account_claims(state(), Db.pubkey(), pagination(), range(), cursor()) ::
+          {:ok, {page_cursor(), [claim()], page_cursor()}} | {:error, reason()}
+  def fetch_account_claims(state, owner_pk, pagination, scope, cursor) do
```

Comment:
Having a hard time following this, what is this function meant to return?

Reply:
It's a list of all NameClaim/AuctionBidClaims an account has made

Reply:
This does not consider accounts that made contract calls to AENS.claim

Reply:
I bealieve it does since some of the name_claim_tx transactions in the field table point to a contract_call transaction. An example of this would be: 

```Elixir
State.mem_state()
|> State.fetch!(
Model.Field,
{:name_claim_tx, nil,
 <<234, 64, 165, 81, 237, 92, 71, 112, 16, 48, 47, 60, 129, 49, 186, 132, 181, 221, 216,
   33, 51, 203, 170, 48, 227, 60, 235, 59, 95, 88, 168, 180>>, 60_678_614}
)
|> then(fn Model.field(index: {_type, _pos_id, _account_id, txi}) ->
    Model.tx(id: hash) = State.fetch!(State.mem_state(), Model.Tx, txi)
    AeMdw.Node.Db.get_tx(hash)
end)
```

Reply:
On that specific case `<<234, 64, 165, 81, 237, 92, 71, 112, 16, 48, 47, 60, 129, 49, 186, 132, 181, 221, 216,
   33, 51, 203, 170, 48, 227, 60, 235, 59, 95, 88, 168, 180>>` is the name hash, not the pubkey of the account that made the claim